### DISCUSSION TITLE
Restart game (fresh)

### DISCUSSION THREAD

**Discussion by tvb** [okaybenji/text-engine#33] [Q&A] [answered]
Hi @okaybenji,

Would it be possible to add a method to restart a game from fresh. Deleting the history/commands and possibly also clearing any local saved game.

**Comment by okaybenji** [comment, marked as answer]
You can simply add a custom command to your game. Stick this in the HTML file:

```
<script>
  const clearScreen = () => document.querySelector('#output').innerHTML = '';
  const reload = () => {
    clearScreen();
    loadDisk();
  };
  commands[0] = Object.assign(commands[0], {reload});
</script>
```

**Comment by tvb** [comment]
Yes, that seems to work 👍🏻
